src/components/Everytime.tsx

```
@@ -3,8 +3,32 @@
+}: {
+  onValidate: (isValid: boolean) => void;
+}) {
+  const [everytimeLink, setEverytimeLink] = useState<string>("");
+  
+  // 에브리 타임 시간표도 유효성을 검사해야 한다면, validate 로 검사중.... 띄우기.
+  
+  useEffect(() => {
+    onValidate(false);
+  }, [])
+
+  const handleEverytimeValidate = useCallback(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: onValidate

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)